managed/services/inventory/services_test.go

```diff
@@ -493,6 +493,123 @@ func TestServices(t *testing.T) {
 		})
 	})
 
+	t.Run("Valkey", func(t *testing.T) {
```

Comment:
Can we add a test that triggers an "invalid service type..." error?

Reply:
In this case, not really since the error is thrown at a higher level (the test tests the individual `AddService` call e.g., `AddValkeyService()`).